browser/components/CodeEditor.js

```diff
@@ -1061,23 +1062,21 @@ export default class CodeEditor extends React.Component {
     } = this.props
     const fontFamily = normalizeEditorFontFamily(this.props.fontFamily)
     const width = this.props.width
-    return (<
-      div className={
+    return (<div
```

Comment:
This is what I've meant in the PR comment. 
```
<div className={...}
  ref='root' ...>
```
Was a problem for `standardx` formatting. But after adding the new line before `className` the formating is OK.
I think that's OK but it was a bit weird during setup because the formatter introduced a linting error.